io.github.garglk.Gargoyle.yaml

```
@@ -0,0 +1,31 @@
+id: io.github.garglk.Gargoyle
+runtime: org.kde.Platform
+runtime_version: "5.15-21.08"
+sdk: org.kde.Sdk
+command: gargoyle
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --socket=system-bus
+  - --filesystem=home
+modules:
+  - name: gargoyle
+    buildsystem: cmake
```

```suggestion
    buildsystem: cmake-ninja
```

this is usually faster.